charts/external-dns/values.yaml

```diff
@@ -126,5 +126,7 @@ provider: aws
 
 extraArgs: []
 
+secrets: {}
```

Comment:
I'd suggest this should be `secretConfiguration` or something singular as `secrets` makes me think multiple secrets.

Reply:
Done